pyccl/halos/profiles_2pt.py

```diff
@@ -60,6 +60,8 @@ def fourier_2pt(self, cosmo, k, M, a, prof, *, prof2=None):
                 second halo profile for which the second-order moment
                 is desired. If ``None``, the assumption is that you want
                 an auto-correlation, and `prof` will be used as `prof2`.
+            diag (bool): If True, both halo profiles depend on the same k. If
+            False, they will depend on k and k', respectively. Default True.
```

Comment:
Can you specify that in this case the output will be 2D (in k and k')?

Reply:
Done